doc-tool/src/dotty/tools/dottydoc/DocDriver.scala

```diff
@@ -33,16 +32,12 @@ class DocDriver extends Driver {
 
   override def newCompiler(implicit ctx: Context): Compiler = new DocCompiler
 
-  def compiledDocs(args: Array[String]): collection.Map[String, Package] = {
```

Comment:
I removed this method since I found no reference for it

Reply:
Nice refactoring! LGTM